src/bundle/Resources/views/themes/admin/account/change_password/form_fields.html.twig

```diff
@@ -32,3 +32,16 @@
         </div>
     </div>
 {%- endblock %}
+
+{%- block form_errors -%}
```

Comment:
Maybe we should create new, more generic file for this kind of blocks? As I assume you ctrl+c ctrl+v it from `admin/content/form_fields.html.twig` file ;)